src/main/java/core/basesyntax/PasswordValidator.java

```diff
@@ -1,7 +1,16 @@
 package core.basesyntax;
 
 public class PasswordValidator {
-    public boolean validate(String password, String repeatPassword) {
-        return false;
+    public static boolean validate(String password, String repeatPassword)
+            throws PasswordValidationException {
+        if (repeatPassword != null && repeatPassword.equals(password)) {
```

Comment:
you don't need else clause at all here, they are redundant and complete validation with a single if condition